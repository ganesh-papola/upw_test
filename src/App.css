

p,
h1,
h2,
h3,
h4,
h5,
h6 {
  margin: 0 !important;
}

input::-webkit-outer-spin-button,
input::-webkit-inner-spin-button {
  -webkit-appearance: none;
  margin: 0;
}

/* Firefox */
input[type="number"] {
  -moz-appearance: textfield;
}
