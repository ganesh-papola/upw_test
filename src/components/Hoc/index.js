export { default as LoaderHoc } from "./LoaderHoc";
