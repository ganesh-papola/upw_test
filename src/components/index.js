export * from "./common";
export * from "./Hoc";
