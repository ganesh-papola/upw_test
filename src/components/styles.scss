@import "../shared/styles/index.scss";

$max-zindex: 1000;
$mid-zindex: 999;
