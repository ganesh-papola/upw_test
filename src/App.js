import "./App.css";
import { Provider } from "react-redux";
import { persistor, store } from "./redux/store";
import Routes from "./routes/routes";
import { PersistGate } from "redux-persist/integration/react";
import { HelmetProvider } from "react-helmet-async";
import { LoaderHoc } from "./components";

function App() {
  return (
    <Provider store={store}>
      <PersistGate persistor={persistor}>
        <HelmetProvider>
          <LoaderHoc>
            <Routes />
          </LoaderHoc>
        </HelmetProvider>
      </PersistGate>
    </Provider>
  );
}

export default App;
