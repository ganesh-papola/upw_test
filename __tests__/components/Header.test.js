import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import { Header } from "../../src/components"
import thunk from 'redux-thunk';
import configureStore from 'redux-mock-store';
import { Provider, useSelector, useDispatch } from 'react-redux';


const middlewares = [thunk];
const mockStore = configureStore(middlewares);
const mockInitialState = {
    auth: {},
    common: {}
}


describe('Header test', () => {
    const props = {}
    it('this is components snapshot rendering test', () => {
        const { container } = render(
            <Provider store={mockStore(mockInitialState)}>
                <Header {...props} />
          </Provider>
        );
        expect(container).toMatchSnapshot();
      });
})
