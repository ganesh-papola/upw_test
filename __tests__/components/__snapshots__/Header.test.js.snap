// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton test this is components snapshot rendering test 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="headerHead"
    >
      <span
        class="headerHeadTitle"
      >
        Dashboard
      </span>
      <div
        class="headerHeadMenu"
      />
    </div>
    <div
      class="headerTabs"
    >
      <div
        class="headerTabsGroup headerTabsLeft"
      >
        <div
          class="headerTabsGroupItems"
        >
          About
        </div>
      </div>
      <div
        class="headerTabsGroup headerTabsRight"
      >
        <div
          class="headerTabsGroupItems"
        >
          Sign Out
        </div>
      </div>
    </div>
  </header>
</div>
`;

exports[`Header test this is components snapshot rendering test 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="headerHead"
    >
      <span
        class="headerHeadTitle"
      >
        Dashboard
      </span>
      <div
        class="headerHeadMenu"
      />
    </div>
    <div
      class="headerTabs"
    >
      <div
        class="headerTabsGroup headerTabsLeft"
      >
        <div
          class="headerTabsGroupItems"
        >
          About
        </div>
      </div>
      <div
        class="headerTabsGroup headerTabsRight"
      >
        <div
          class="headerTabsGroupItems"
        >
          Sign Out
        </div>
      </div>
    </div>
  </header>
</div>
`;
